spec/unit_tests/moab/file_group_difference_spec.rb

```diff
@@ -232,46 +232,30 @@
     expect(new_diff.added).to eq 1
   end
 
+  let(:matching_keys_signatures) do
+    new_diff.matching_keys(v1_content.signature_hash, v3_content.signature_hash)
+    # page-2.jpg, page-3.jpg, title.jpg
+  end
+
   specify '#tabulate_unchanged_files' do
-    basis_group = v1_content
-    other_group = v3_content
-    signatures = new_diff.matching_keys(basis_group.signature_hash, other_group.signature_hash)
-    expected_sig_fixity1 = {
-      size: "39450",
-      md5: "82fc107c88446a3119a51a8663d1e955",
-      sha1: "d0857baa307a2e9efff42467b5abd4e1cf40fcd5",
-      sha256: "235de16df4804858aefb7690baf593fb572d64bb6875ec522a4eea1f4189b5f0"
-    }
-    expected_sig_fixity2 = {
-      size: "19125",
-      md5: "a5099878de7e2e064432d6df44ca8827",
-      sha1: "c0ccac433cf02a6cee89c14f9ba6072a184447a2",
-      sha256: "7bd120459eff0ecd21df94271e5c14771bfca5137d1dd74117b6a37123dfe271"
-    }
-    expected_sig_fixity3 = {
-      size: "40873",
-      md5: "1a726cd7963bd6d3ceb10a8c353ec166",
-      sha1: "583220e0572640abcd3ddd97393d224e8053a6ad",
-      sha256: "8b0cee693a3cf93cf85220dd67c5dc017a7edcdb59cde8fa7b7f697be162b0c5"
-    }
-    expect(signatures.collect { |s| s.fixity }).to eq [expected_sig_fixity1, expected_sig_fixity2, expected_sig_fixity3]
```

Comment:
note that this entire section, L236-L257, is doing setup and saying "look, here are the original fixities".  I found it more useful to name the expected fixities appropriately within the tests, and indicate the files expected to be part of the signature group in the "let" statement (for matching_keys_signatures)